src/Tasks/AssemblyDependency/ReferenceTable.cs

```diff
@@ -3014,7 +3014,7 @@ internal void LogHigherVersionUnresolveDueToAttribute(bool displayPrimaryReferen
         {
             if (displayPrimaryReferenceMessage)
             {
-                _log.LogWarningWithCodeFromResources("ResolveAssemblyReference.PrimaryReferenceOutsideOfFrameworkUsingAttribute", reference.PrimarySourceItem.ItemSpec /* primary item spec*/, reference.FrameworkNameAttribute /*Version of dependent assemby*/, targetedFramework);
+                _log.LogErrorWithCodeFromResources("ResolveAssemblyReference.PrimaryReferenceOutsideOfFrameworkUsingAttribute", reference.PrimarySourceItem.ItemSpec /* primary item spec*/, reference.FrameworkNameAttribute /*Version of dependent assemby*/, targetedFramework);
```

Comment:
Can you please add a unit for this, if there's no failing ones?

Reply:
@cdmihai thanks for the feedback!

OK, I will add one unit test for this.